components/tel/Core/Src/main.c

```diff
@@ -196,6 +199,8 @@ void HAL_TIM_PeriodElapsedCallback(TIM_HandleTypeDef *htim)
   if (htim->Instance == TIM1) {
     HAL_IncTick();
   }
+
+  CPU_LOAD_timer_overflow_handler(htim);

```

Comment:
This is within the "Period Elapsed Callback", so this function gets called when a timer has overflow/reset. My function just checks if this function is being called for the timer that has been provided to the CPU Load software component and if it is it will increment the overflow counter.

Reply:
Period is the same as amount of ticks